tests/integration/session_management/test_niscope_reservation.py

```diff
@@ -83,6 +83,24 @@ def test___sessions_created___get_niscope_connections___returns_connections(
         ]
 
 
+def test___no_pin_map___session_created___get_niscope_connection___returns_connection(
```

Comment:
Can we consider using [pytest.mark.xfail](https://docs.pytest.org/en/stable/how-to/skipping.html#xfail-mark-test-functions-as-expected-to-fail) decorator to temporarily ignore this test failure if we're not planning to update the `measlinkbot` with the latest changes any sooner?

Reply:
We can do that if necessary. @MounikaBattu17 which infrastructure-as-code file(s) determine which version of MeasurementLink is being used? Will this be uptaken automatically or do we need to periodically / as needed update the version?

Reply:
For the multiplexer feature, I updated the [driver_feeds.win.yml](https://dev.azure.com/ni/DevCentral/_git/ni-central?path=/eng/iac/cfg_mgmt/salt/srv/salt/measlinkbot/win/resources/driver_feeds_win.yml&version=GBusers/jay/update-win-boot-timing&line=4&lineEnd=5&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents) and deployed the bot to encompass the necessary multiplexer support updates. The same was done for the pin group feature as well.

Related PRs:
- [Pull Request 660173](https://dev.azure.com/ni/DevCentral/_git/ni-central/pullrequest/660173): measlinkbot: update MeasurementLink version
- [Pull Request 666346](https://dev.azure.com/ni/DevCentral/_git/ni-central/pullrequest/666346): measlinkbot: update MeasurementLink version

But now I see that we must start using the ni-instrumentstudio feed for the bot.

Reply:
> We can do that if necessary. @MounikaBattu17 which infrastructure-as-code file(s) determine which version of MeasurementLink is being used? Will this be uptaken automatically or do we need to periodically / as needed update the version?

No, MeasurementLink version is not automatically updated, it is a manual process. The version needed should be updated [here](https://dev.azure.com/ni/DevCentral/_git/ni-central?path=/eng/iac/cfg_mgmt/salt/srv/salt/measlinkbot/win/resources/driver_feeds_win.yml&version=GBmain&line=5&lineEnd=6&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents) and this [pipeline](https://dev.azure.com/ni/DevCentral/_build?definitionId=13829&_a=summary) should be built and deployed with the updated changes to get reflected in this GitHub pipeline. Kindly let me know for any clarifications.